gradle.properties

```diff
@@ -1,3 +1,3 @@
-version=1.0
-description=EPAM Report Portal. Gitlab plugin.
-pluginId = Gitlab
+version=23.3
```

Comment:
All our plugins and services should follow semantic versioning.
So please use version 1.0.